system/edit-config.in

```diff
@@ -33,6 +37,7 @@ EOF
 
   cd "${NETDATA_STOCK_CONFIG_DIR}" || exit 1
   ls >&2 -R ./*.conf ./*/*.conf
+  make_it_safer()
```

Comment:
```suggestion
  make_it_safer
```

Reply:
If this seems so important to you, I can omit the parentheses.

Reply:
It doesn't work on my Deb11 VM with parentheses.

Reply:
To be portable it does indeed need to not have parentheses here.